groups.go

```diff
@@ -420,6 +448,7 @@ func (s *GroupsService) TransferSubGroup(gid interface{}, opt *TransferSubGroupO
 type UpdateGroupOptions struct {
 	Name                                 *string                     `url:"name,omitempty" json:"name,omitempty"`
 	Path                                 *string                     `url:"path,omitempty" json:"path,omitempty"`
+	Avatar                               *GroupAvatar                `url:"-" json:"avatar,omitempty"`
```

Comment:
Shouldn't this be:
```suggestion
	Avatar                               *GroupAvatar                `url:"-" json:"-"`
```

Reply:
hmmm, good question. I think in case of `UploadRequest()` the options are any ways never JSON encoded, but just form fields ... So, probably it doesn't matter. I have to double check.